pkg/dependency/credhelper/cred_helper.go

```diff
@@ -96,3 +103,64 @@ func newDeps(
 
 	return deps
 }
+
+// RefreshConfigFile refreshes config.json according to finch.yaml.
+func RefreshConfigFile(fs afero.Fs, logger flog.Logger, finchConfigPath, configJSONPath string) error {
+	systemDeps := system.NewStdLib()
+	mem := fmemory.NewMemory()
+
+	finchCfg, err := config.LoadFinchConfig(fs, finchConfigPath, logger, systemDeps, mem)
```

Comment:
Same for config, we can pass it as a dependency to the vm commands, and then to this function instead of loading it again

Reply:
@pendo324 

Does this mean that the `fc` being passed in the following is passed to `newVirtualMachineCommand()`, which in turn is passed to `newStartVMCommand()` and `newInitVMCommand()`, to avoid re-loading the config? 

- https://github.com/runfinch/finch/blob/main/cmd/finch/virtual_machine.go#L105